kernel/kernel-util/src/main/java/org/sakaiproject/util/PasswordCheck.java

```diff
@@ -256,5 +256,46 @@ public static boolean isAcceptableLength(String passwd, int min, int max) {
 		return false;
 	}
 	
-	
+	/**
+	 * Generate a 9 digit password. Chars are randomly picked from lower-case/upper-case alpha lists, a numerical list, and a symbol list
+	 * @return 
+	 */
+	public static String generatePassword()
```

Comment:
I think it could be useful to have a generatePassword(int length) rather than just have it default to 9. Since only 2 places use it this should be easy to change. You could have a default no op method that defaults to 9 as well.